style.css

```diff
@@ -134,3 +152,98 @@
   margin-top: 2rem;
   flex-wrap: wrap;
 }
+
+span {
+  font-size: 3rem;
+  text-align: center;
+  color: #c8c8d4;
+  margin-top: 3rem;
+}
+
+.works-flex {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  margin-left: 6.4vw;
+  margin-top: 2rem;
+  margin-bottom: 2rem;
+}
```

Comment:
This content should be laid out using `grid`.

<img width="370" alt="Screen Shot 2022-07-07 at 21 39 14" src="https://user-images.githubusercontent.com/82426682/177905948-d5498d14-06de-4870-a439-71d02c3c3589.png">

The section is missing the background image as well. Please refer to the Figma template and add the appropriate background image.

<img width="708" alt="Screen Shot 2022-07-07 at 21 40 27" src="https://user-images.githubusercontent.com/82426682/177906111-728fb8c3-f3cd-4308-a382-390e334d40f8.png">

Reply:
The main container of the projects uses display:grid now.